proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+<lynX> Patents also have their useful aspects. I would suggest to work on societal taxes and dividends for redistribution, this way the gains the corps had through public research flows back to the citizen. See also 'European New Deal'.
+
+As we can see, there is a trend towards big monopoly power within the tech industry: Google, Microsoft, Apple, Facebook, Amazon. The reason for this is, that with these new technologies cooperation is a much more efficient way of production then competition. So these large corporations have the advantage of being able to have cooperation within their company. 
+<lynX> This is not an accurate description on how the big 5 became monopolies.
+
+Of course the monopoly power is extremely harmful and dangerous. Thus we need to find ways to promote cooperative production without the need for mega-corporation. One extremely successful model here is open-source/free software: It encourages cooperation: while everyone can benefit from the existing base of software, the model also encourages the contribution to this pool.
+<lynX> Free/Open Source software was one of the winning ingredients of Google and subsequent monopolists. When it came to making a product they moved the code to the private realm, just like it is described with research above. FLOSS alone is not the solution, it needs supporting legislation.
+
+Free/Open Source software is also the only way to guarantee real security on the net: Only when the code is open to be studied by everyone can we be sure that there are no back-doors hidden in the code.Thus we need to foster the model of Free Software/Open Source and also try to apply it to other fields of science and technology. With the rise of 3D-Printing and other computerized technologies, the realm of physical production becomes more and more similar to that of software production. Thus the same rules make sense there.Political demands here would be:
+
+* Make Free Software/Open Source mandatory for all public institutions and publicly funded projects.
+* Public financing of strategic free software/open-source projects.
+* Demand "Open Access" for all scientific research.
+* Abolish the patent system. (Gradually removing the scope and lenght of patent zero). Instead we need to gradually replace it with a system, where we demand that all products sold need to come with full receipts, source code, descriptions of production processes, etc. 
```

Comment:
But: corporate researchs will need to be rewarded as well – Here we should encourage a different patent system - subliminal guy

Reply:
One where subsidies are only granted if all research is public? :) - Anonymous